test/MeshKernelNETTest/Api/MeshKernelTest.cs

```diff
@@ -2493,5 +2493,56 @@ public void Mesh2dMakeGlobaThroughApi()
                 }
             }
         }
+
+
+        [Test]
+        public void Mesh2dUndoRedoThroughApi()

```

Comment:
Redo is not tested here. I propose to use a longer name that explains that two delete node calls are undone.